braze_user_csv_import/app.py

```diff
@@ -295,17 +295,20 @@ def _process_value(
     if cast == str:
         return value
 
+    if cast:
+        return cast(_process_value(value))
+
     stripped = value.strip().lower()
     leading_zero_int = len(stripped) > 1 and stripped.startswith('0') \
         and not stripped.startswith('0.')
-    if cast:
-        return cast(_process_value(value))
+
+    is_numeric = stripped.replace('.', '').replace('-', '').isdigit()
```

Comment:
do you want to only replace the - if it's the first character? otherwise i think 123-45 might be true. 
Also might want to replace '.' once ie ('.','',1) otherwise you might allow for 111.111.111 ?
Or are all these checks don via _is_int or _is_float ?

Reply:
Correct, you can check out test cases. Those examples will still fail on `_is_float/int` and they won't be cast.

Reply:
https://github.com/braze-inc/growth-shares-lambda-user-csv-import/blob/master/tests/test_app.py#L118